src/components/DataSources/index.js

```diff
@@ -0,0 +1,72 @@
+import { Grid, Typography } from "@material-ui/core";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React, { useState, useEffect } from "react";
+
+import useStyles from "./useStyles";
+
+import Header from "@/pesayetu/components/Header";
+import Link from "@/pesayetu/components/Link";
+import Section from "@/pesayetu/components/Section";
+
+const DataSources = ({ items, title, subtitle, ...props }) => {
+  const classes = useStyles(props);
+  const [selectedIndex, setSelectedIndex] = useState(0);
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setSelectedIndex((selectedIndex + 1) % items?.length);
+    }, 1000);
+    return () => clearInterval(interval);
+  });
```

Comment:
Also confused what you're trying to do here

Reply:
so I expect each resource added from the cms to include a cover page image, and what I do is to iterate over them in intervals of 3 secs showing the cover page of each link. @KhadijaMahanga cc @kilemensi

Reply:
🤔 ... is this in the XDs?

Reply:
the XD seems confusing there, the report images is just one of the datasources. @kilemensi